tests/testShapeletPsfModels.py

```diff
@@ -47,18 +48,25 @@ def setUp(self):
         self.schema = lsst.afw.table.SourceTable.makeMinimalSchema()
         self.centroidKey = lsst.afw.table.Point2DKey.addFields(self.schema, "centroid", "centroid", "pixels")
         self.schema.getAliasMap().set("slot_Centroid", "centroid")
+        self.psfDir = os.path.join(os.environ["MEAS_MODELFIT_DIR"], "tests", "data", "psfs")
 
     def tearDown(self):
         del self.exposure
         del self.schema
         del self.centroidKey
-
-    def makePsf(self, psfname):
-        data = lsst.afw.image.ImageF(psfname).getArray().astype(numpy.float64)
+        del self.psfDir
+
+    def makePsf(self, psfname, max=None):
+        data = lsst.afw.image.ImageF(os.path.join(self.psfDir, psfname)).getArray().astype(numpy.float64)
+        if not max is None:
```

Comment:
`if max is not None` is much preferred stylistically here, even though they do the same thing (http://stackoverflow.com/questions/2710940/python-if-x-is-not-none-or-if-not-x-is-none).